pkgs/stdenv/generic/setup.sh

```diff
@@ -422,50 +421,55 @@ fi
 
 
 substitute() {
-    local input="$1"
-    local output="$2"
+    local input=$1
+    local output=$2
```

Comment:
I could leave as is, but I figured given it didn't matter I should pick one and be consistent. I could normalize the other way if you like.